test/unit/test_configcommon.py

```diff
@@ -124,3 +125,38 @@ def test_clean_dynamic_variables(autosubmit_config: Callable) -> None:
 
     assert len(as_conf.dynamic_variables) == 1
     assert 'jaspion_eats' in as_conf.dynamic_variables
+
+
+def test_yaml_deprecation_warning(tmp_path, autosubmit_config: Callable):
+    """Test that the conversion from YAML to INI works as expected, without warnings.
+
+    +    Creates a dummy AS3 INI file, calls ``AutosubmitConfig.ini_to_yaml``, and
+    +    verifies that the YAML files exists and is not empty, and a backup file was
+    +    created. All without warnings being raised (i.e. they were suppressed).
+    +    """
```

Comment:
These `+` signs are probably from the Git diff :grimacing: They need to be removed...